### DISCUSSION TITLE
GitHub Education Application keep getting rejected

### DISCUSSION THREAD

**Discussion by Nguyenhoangdung335** [community/community#142684] [GitHub Education] [answered]
### Hi everyone,

I am currently a College student in Vietnam and I am trying to sign up for the Student Education Pack. However, I keep getting rejected just after submitting the application with the reasons in the image 
![Screenshot 2024-10-26 171608](https://github.com/user-attachments/assets/b3282105-d952-4e9b-a198-23aab6488e86)
My student ID card does have the school name as well as the logo with my name and the valid date. I have also filled the billing information and completed my profile. Is there anything wrong with my profile.

**Comment by DevKyLeOfficial** [comment]
In my opinion, you need to upload documents using the information provided and also you need to be coordinated to your school for the acknowledgement of the information you need to

**Comment by HosamShaban** [comment]
Check Photo Quality and Details

Use a School Email Address

Update Your GitHub Profile

Provide Additional Supporting Documents

Submit a Detailed Explanation

**Reply by anuj-876** [reply]
I think U should check the resolution and image size and u should have done with 2f-authentication and last create the billing with exact name uses on the documents u will be sharing

**Reply by Nguyenhoangdung335** [reply]
I don't know if it is because of the resolution or the photo quality. I am using my phone camera to take the photo with both the landscape and portrait orientation. I also already have 2 factor authentication on my account so it wouldn't be because of that. As for the billing info, I don't know where should I put my name. In Vietnam, the first name is at the end so I don't know if I should put it as first name or last name.

**Reply by HosamShaban** [reply]
Enter your given name as "First Name" and your family name as "Last Name."

**Reply by Nguyenhoangdung335** [reply]
I just try putting my given name as First name and family and middle name as last name, but still doesn't work. 
Currently on my id, my name is Nguyễn Hoàng Dũng, while in my billing info it is Dũng Nguyễn Hoàng. Would it still work or do I have to provide some kinds of translated text without the regional symbol.

**Reply by HosamShaban** [reply]
Given Name: "Hoang Dung"
Family Name: "Nguyen"

**Reply by Nguyenhoangdung335** [reply]
Now it give me this after I submitted, do I need to include a translated ID with only English in my photo
![image](https://github.com/user-attachments/assets/c7692674-49bd-4c39-8563-d45898cce18f)

**Reply by HosamShaban** [reply]
translate English if needed.

Match Your GitHub Profile Name with the Document.

**Reply by Nguyenhoangdung335** [reply, marked as answer]
It finally got approved . I try changing my profile name into English without the symbols, which seems to solve the third problem. I then go on to the mobile web version to make the application instead of my computer. As for the photo, I included a hand-written English translated text beside the ID which seems to do the trick.

**Reply by HosamShaban** [reply]
Great

**Reply by HosamShaban** [reply]
Make marked answer and answered

**Comment by anuj-876** [comment]
On Sat, 26 Oct 2024 at 10:01 PM, Nguyen Hoang Dung ***@***.***>
wrote:

> Closed #142684 <https://github.com/orgs/community/discussions/142684> as
> resolved.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/142684>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7PARC4UYNWFJY7K7JBKGTLZ5O7WTAVCNFSM6AAAAABQUVPM3OVHI2DSMVQWIX3LMV45UABFIRUXGY3VONZWS33OIV3GK3TUHI5E433UNFTGSY3BORUW63R3GE2TONBRGE3A>
> .
> You are receiving this because you commented.Message ID:
> <community/community/repo-discussions/142684/discussion_event/1574116@
> github.com>
>
Put your name as per college documents thats it
